docs/fundamentals/toc.yml

```
@@ -645,6 +645,9 @@ items:
       items:
       - name: BinaryFormatter security guide
         href: ../standard/serialization/binaryformatter-security-guide.md
+      - name: BinaryFormatter event source
+        href: ../standard/serialization/binaryformatter-event-source.md
+  - name: System.CommandLine
```

Looks like System.CommandLine got duplicated.

```suggestion
      - name: BinaryFormatter event source
        href: ../standard/serialization/binaryformatter-event-source.md
```

---

Good catch @tdykstra!